### PR TITLE
LLM: Add gguf mistral model support

### PR DESCRIPTION
**Pull request by hzjane**
## Description
LLM: Add gguf mistral model support

### 1. Why the change?
Add gguf mistral model support

### 2. User API changes
No.

### 3. Summary of the change 
Can support load  mistral gguf model.

### 4. How to test?
- [ ] N/A
- [ ] Unit test
- [x] Application test
- [ ] Document test
- [ ] ...

### CODE REVIEW

File: python/llm/src/bigdl/llm/transformers/gguf/api.py [hunk 1]
```diff
@@ -39,9 +39,14 @@ def load_gguf_model(fpath: str, dtype: torch.dtype = torch.float):
 
     with torch.no_grad():
         if model_family == "llama":
-            from .models.llama import load_gguf_llama
-
-            model, tokenizer = load_gguf_llama(loader, dtype)
+            model_name = loader.config["general.name"]
+            if "mistral" in model_name:
```
Comment (glorysdj) [inline python/llm/src/bigdl/llm/transformers/gguf/api.py:43]: to lower for model_name

**Review by glorysdj — APPROVED** [review]
LGTM

**Comment by jason-dai** [comment]
In the example readme:

1) Remove `Note: Only LLaMA2 family models are currently supported`
2) Add model list that have been verified

**Comment by hzjane** [comment]
> In the example readme:
> 
> 1. Remove `Note: Only LLaMA2 family models are currently supported`
> 2. Add model list that have been verified

I did it in another pr.

**Pull request merged**